build/storage/tests/it/ptf_tests/system_tools/test_platform.py

```diff
@@ -115,46 +113,145 @@ def set_system_setup(self):
         if not self._is_docker():
             self._install_docker()
 
+    def get_pid_from_port(self, port: int):
+        return self.terminal.execute(
+            f"sudo netstat -anop | grep -Po ':{port}\s.*LISTEN.*?\K\d+(?=/)' || true"
+        )
 
-class StorageTargetPlatform(BaseTestPlatform):
+    def kill_process_from_port(self, port: int):
+        """Raise error if there is no process in specific port"""
+        pid = self.get_pid_from_port(port)
+        self.terminal.execute(f"sudo kill -9 {pid}")
 
-    def __init__(self):
-        config = StorageTargetConfig()
-        terminal = SSHTerminal(config)
-        super().__init__(terminal)
+    @retry(stop=stop_after_delay(180), reraise=True)
+    def wait_for_running(self, image_name):
+        out = self.terminal.execute("docker ps")
+        if image_name not in out:
+            raise Exception("Container is not running")
 
-    # TODO add implementation
+
+class StorageTargetPlatform(BaseTestPlatform):
     def create_subsystem(self):
-        pass
+        cmd_sender_id = (
+            self.terminal.docker.get_docker_containers_id_from_docker_image_name(
+                "cmd-sender"
```

Comment:
Do I understand correctly, that you need that `cmd-sender` container to send commands to other containers since you do not have required tools on your local platform?

So you can proceed with containerization of these tests, it won't take much time

Reply:
future task